tuna/dbBase/base_class.py

```diff
@@ -34,23 +36,26 @@
 class BASE():
   """Base class for our own common functionalities among tables"""
 
-  __table_args__ = {'mysql_engine': 'InnoDB'}
-  __mapper_args__ = {'always_refresh': True}
+  __table__: Column = None
+  __table_args__: Dict[str, str] = {'mysql_engine': 'InnoDB'}
+  __mapper_args__: Dict[str, bool] = {'always_refresh': True}
 
   id = Column(Integer, primary_key=True)
   insert_ts = Column(DateTime, nullable=False, server_default=sqla_func.now())
   update_ts = Column(
       DateTime,
       nullable=False,
       server_default=text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'))
-  valid = Column(TINYINT(1), nullable=False, server_default="1")
+  valid: Column = Column(TINYINT(1), nullable=False, server_default="1")
```

Comment:
Please match type schema with previous fields. My preference is removing the typing, because it is visible by the constructor.